app/views/projects/index.html.erb

```diff
@@ -48,6 +48,18 @@
           <%= project.address %><br>
         <% end %>
         <div>Cost: <%= project.cost_string %></div>
+        <div>Created by: <%= project.user.name %></div>
+        <% if project.vote_count > 0 %>
+          <div>Voted by:
+            <% Vote.where(project_id: project.id).each_with_index do |vote, index| %>
```

Comment:
`where` probably shouldn't be in views. It would be better if we could do something like `project.votes.each do |vote|`. I'm not sure what changes have to be made to make that works.